app/controllers/package_set.js

```diff
@@ -36,27 +38,30 @@ export default Controller.extend({
   allPackages: computed(
     "model",
     "model.isSet",
-    "model.packagesAndSets.@each.isAvailable",
+    "model.packagesAndSets@each.isAvailable",
     function() {
       var record = this.get("model");
       if (!record) {
         this.send("noItemsPresent");
+        this.set("packageUnavailableInSet", true);
         return [];
       }
 
-      return record.get("isSet")
-        ? record.get("packages").filterBy("isAvailable")
-        : [record];
+      return record.get("isSet") ? record.get("packages") : [record];
     }
   ),
 
-  notAvailableInStock: computed(
+  notAvailableInStock: observer(
```

Comment:
can we avoid observer?

cc: @abulsayyad123

Reply:
In order to fix regression of live updating I have added observer which was not happening with computed..but please do let me know if there is any way to do it